core/sdram_startup.h

```diff
@@ -0,0 +1,519 @@
+#include <stdint.h>
```

Comment:
Is this file used anymore? Or just vestigial from when we wanted to move SDRAM to startup.

Reply:
Yeah it can be removed.